scripts/jtsync/README.md

```diff
@@ -0,0 +1,57 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+          --ci SERVICE                 Which ci is used (suse or opensuse)
```

Comment:
Since this is a switch parameter there is either "suse" or "opensuse" not "openSUSE".